### DISCUSSION TITLE
How do I fix this error?

### DISCUSSION THREAD

**Discussion by Naomi-Faith** [dbeaver/dbeaver#18286] [Q&A]
SQL Error [3524] [42000]: [Teradata Database] [TeraJDBC 17.10.00.23] [Error 3524] [SQLState 42000] The user does not have CREATE TABLE access to database DWU_RBTS_RPS_PRC

Thank you for your time.

**Comment by LonwoLonwo** [comment]
Hello. You need permissions for table creation in the database DWU_RBTS_RPS_PRC.
You probably need to execute something like this:

`GRANT CREATE ON DATABASE "DWU_RBTS_RPS_PRC" TO user;`

You can read more here: https://docs.teradata.com/r/Teradata-Database-SQL-Data-Control-Language/June-2017/Statement-Syntax/GRANT-SQL-Form

**Comment by Naomi-Faith** [comment]
Thank you so much, I did find something like that in the historical files.

Happy Thanksgiving!


Regards

**Comment by LonwoLonwo** [comment]
Happy Thanksgiving!
